embassy-rp/src/dma.rs

```diff
@@ -174,6 +174,19 @@ impl<'a, C: Channel> Transfer<'a, C> {
 
         Self { channel }
     }
+    /// Abort a DMA transfer early
+    ///
+    /// Returns the count of transfers still left to do and the data size of the transfers
+    pub fn abort(self) -> u32 {
```

Comment:
one last nit, sorry :sweat_smile: . could you change this to return usize? so the user doesn't have to cast if they want to do index math or slice stuff.

Reply:
also docs need updating.

Reply:
All done (rebase as well)